theme/static/style/style.css

```diff
@@ -177,6 +177,12 @@ nav button {
     margin-bottom: 2rem;
 }
 
+/* make blog posts' hero images responsive */
+.article p:first-child img {
+    width: 100%;
```

Comment:
Comment out `width` b/c if the first image is smaller it gets stretched, for example here:
http://kiwitcms.org/blog/kiwi-tcms-team/2020/04/09/please-vote-for-kiwi-tcms-in-2020-eu-web-awards/